src/components/CreateResetPassword/CreateResetPassword.js

```diff
@@ -96,13 +131,17 @@ class CreateResetPassword extends Component {
         newPassword,
         confirmPassword,
       },
+      token,
     } = this.props;
     const { passwordMasked } = this.state;
     const submissionStatus = submitting || submitSucceeded;
     const buttonDisabled = !isEmpty(errors) || submissionStatus || !(newPassword && confirmPassword);
-    const buttonLabelId = `stripes-core.${submissionStatus ? 'settingPassword' : 'setPassword'}`;
     const passwordType = passwordMasked ? 'password' : 'text';
-    const passwordToggleLabelId = `stripes-core.button.${passwordMasked ? 'show' : 'hide'}Password`;
+    const buttonLabelId = `${this.translationNamespaces.module}.${submissionStatus ? 'settingPassword' : 'setPassword'}`;
+    const passwordToggleLabelId = `${this.translationNamespaces.button}.${passwordMasked ? 'show' : 'hide'}Password`;
+
+    // Todo don't have a back-end yet, should be parsed from token
+    const username = 'diku_admin';
```

Comment:
What side-effects does this have for someone logged in with a user other than diku_admin?  I am weary of hard-coding a username like this even if it is expected to be temporary.  Not everyone may use diku_admin for development and other usernames are certainly valid in demo/live environments.

Reply:
This variable currently is used for validation rule that checks that password doesn't includes username. So I think it doesn't have any side-effects.